### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by totop716**
- Update manifest.json
- Remove service worker
- Remove unnecessary packages

### CODE REVIEW

File: src/pages/AnalyticsPage/AnalyticsLiquidityChart.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ const AnalyticsLiquidityChart: React.FC = () => {
       )}
       <Box>
         <Typography style={{ color: palette.text.disabled }} variant='caption'>
-          {moment().format('MMM DD, YYYY')}
+          {dayjs().format('MMM DD, YYYY')}
```
Comment (FriskyHamTitz) [inline src/pages/AnalyticsPage/AnalyticsLiquidityChart.tsx:115]: Why did you remove moment for dayjs?

File: src/components/AreaChart/AreaChart.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { createStyles, makeStyles } from '@material-ui/core/styles';
 import { Box, Typography } from '@material-ui/core';
 import Chart from 'react-apexcharts';
-import moment from 'moment';
+import dayjs from 'dayjs';
```
Comment (FriskyHamTitz) [inline src/components/AreaChart/AreaChart.tsx:5]: Why? I understand dayJs is suppose to be a minifed version of moment but it doesn't have the same credibility of the original package.

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/components/AreaChart/AreaChart.tsx (hunk 2 above)
Comment (totop716) [reply src/components/AreaChart/AreaChart.tsx:5]: I think we don't need to use both of moment and dayjs because dayjs has all functions which moment has. Tested on my side and was able to get all values we got using moment with dayjs.

File: src/pages/AnalyticsPage/AnalyticsLiquidityChart.tsx (hunk 1 above)
Comment (totop716) [reply src/pages/AnalyticsPage/AnalyticsLiquidityChart.tsx:115]: I removed moment as dayjs has all functionalities that moment has

**Review by FriskyHamTitz — APPROVED** [review]

File: public/manifest.json [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
-      "sizes": "192x192"
+      "purpose": "any maskable"
     },
     {
-      "src": "logo512.png",
+      "src": "./logo_circle.png",
+      "sizes": "50x50",
       "type": "image/png",
-      "sizes": "512x512"
+      "purpose": "any maskable"
```
Comment (FriskyHamTitz) [inline public/manifest.json:15]: What's this for?

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: public/manifest.json (hunk 3 above)
Comment (totop716) [reply public/manifest.json:15]: Manifest was loading the invalid images and it returns 404 error. I updated manifest to fix the error.

**Review by FriskyHamTitz — APPROVED** [review]

**Review by 3mp8r3 — APPROVED** [review]
Looks good to merge!

**Review by sameepsi — APPROVED** [review]
Looks good to me
